app/youtube-grabber.js

```diff
@@ -58,17 +61,22 @@ class YoutubeGrabber {
     const channelMetaData = channelPageDataResponse.metadata.channelMetadataRenderer
     const channelHeaderData = channelPageDataResponse.header.c4TabbedHeaderRenderer
     const headerTabs = channelPageDataResponse.contents.twoColumnBrowseResultsRenderer.tabs
-
+    const channelTabs = headerTabs.filter(tab => {
+      return typeof tab.tabRenderer !== 'undefined'
+    }).map(tab => {
+      return tab.tabRenderer.title
+    })
```

Comment:
I guess we don't want `null` as well?
```suggestion
    const channelTabs = headerTabs.filter(tab => tab.tabRenderer != null).map(tab => tab.tabRenderer.title)
```

Reply:
eslint rule requires strict equals so I can't commit your suggestion exactly but i can add a null check

Reply:
Ya I think eslint rule should be updated to exclude `var != null` it's so common...